### ISSUE TITLE
Android Version

### ISSUE THREAD

**Issue by ianclawson** [Stars2Apples/Support#1]
## TL;DR:

**There are currently (and regretfully) no plans to create an Android version of Stars 2 Apples.**

But vote on this issue with a 👍 if you want an Android version, and if there are enough votes, then plans may change.

## Explanation:

Building an Android version from scratch requires much more effort than I am able to give, unfortunately. Stars 2 Apples is created and maintained entirely by one developer (me). Between the demands of everyday life, a busy day job, trying to be a good husband to my wife and be a good father to our children, carving out time to work on Stars 2 Apples is tricky. Whatever time I am able to give would be better spent adding features and fixing bugs for the current iOS app.

## Longer Explanation:

When we first created Stars 2 Apples, we had no idea if anyone would download it. It began as this little app that I created to help my wife teach her online ESL classes. We were amazed with the response! We immediately dreamed up plans for how we could make the app better, chief among which was a plan to create an Android version of the application. We knew it would take time to create from scratch, but we heard from so many people that teachers wanted it, so we put it on our roadmap.

What we didn't realize at the time was how much time maintaining the iOS version alone would take. It became more important to fix bugs and add features to better support our current teachers rather than try to create a new platform for teachers who wanted an Android version but aren't currently using the app.

The other major thing that happened was that as time went on, I began to have less and less time to dedicate to the app. When we first built S2A we had one child, who was a baby at the time. We now have three children. Between the demands of family life + a demanding full time day job, whatever free time I can get to work on projects like Stars 2 Apples I have to fight desperately for, so it's better spent improving the iOS version.

## Will this ever change?

Unfortunately, this is something I cannot commit to right now. Mainly because of the time constraints previously described. 

However, another major reason is that we simply have no idea how many people are even interested in an Android version in the first place. Anecdotally we have hard from ~30 people or so that they would love an Android version. If we found out the actual number of teachers is much, much higher than that, then that becomes harder to say no to, and would make the effort required more worth it.

### *If you are interested in having an Android version of Stars 2 Apples, please let me know by leaving a 👍 on this issue so I can track it. Thank you.*
